.github/workflows/ci.yml

```diff
@@ -461,12 +465,16 @@ jobs:
             version: "24.04"
           - distro: ubuntu
             version: "25.04"
+          - distro: ubuntu
+            version: "25.10"
           - distro: debian
             version: bullseye
           - distro: debian
             version: bookworm
           - distro: debian
             version: trixie
+          - distro: debian
+            version: forky
```

Comment:
Now that the github CI accepts anchors, what about using them to avoid redefining the matrix in multiple locations?